django/forms/fields.py

```diff
@@ -750,7 +750,7 @@ class ChoiceField(Field):
         'invalid_choice': _('Select a valid choice. %(value)s is not one of the available choices.'),
     }
 
-    def __init__(self, choices=(), **kwargs):
+    def __init__(self, *, choices=(), **kwargs):
```

Comment:
Your proposal in https://github.com/django/django/pull/8590 looks good. I proposed https://github.com/django/django/pull/8592 to address my other comments.